common/src/main/java/org/opensearch/sdk/DeleteDataObjectRequest.java

```diff
@@ -41,12 +44,17 @@ public String id() {
         return this.id;
     }
 
+    public String tenantId() {
```

Comment:
Needs a javadoc.

Won't repeat this but almost everything added here has no javadocs.